R/Spectra.R

```diff
@@ -1684,7 +1692,8 @@ setMethod("filterIntensity", "Spectra",
                            "a lower and upper limit.")
                   object <- addProcessing(object, .peaks_filter_intensity,
                                           intensity = intensity,
-                                          msLevel = msLevel.)
+                                          msLevel = msLevel.,
```

Comment:
Do we need to keep this? Can't we define that the default `spectraVariables = c("msLevel", "centroided")` and drop those single variable arguments? I see that `msLevel` shows up sometimes, but not `centroided`, so I might be missing something.

Reply:
Or, does `msLevel` refer to the MS levels on which the processing need to be applied? Then it makes sense.

Reply:
Right, `msLevel` refers to the MS level on which the processing need to be applied. I know, this thing with `msLevel` and `spectrumMsLevel` is confusing...